src/components/NewArrivals/NewArrivals.tsx

```diff
@@ -1,50 +1,29 @@
-"use client";
-import { useEffect, useState } from "react";
-import { fetchProducts } from "@/api/api";
 import { ProductCard } from "../ProductCard/ProductCard";
 import styles from "./NewArrivals.module.scss";
-import { ThreeDots } from "react-loader-spinner";
+import { ProductsProps } from "@/types/ProductsProps";
```

Comment:
try to be consistent in using either relative or absolute paths in your import statements. absolute path si always beter ofc